website/docs/r/google_project_default_service_accounts.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_project_default_service_accounts"
+sidebar_current: "docs-google-project-default-service-accounts-x"
+description: |-
+  Allows management of Google Cloud Platform project default service accounts.
+---
+
+# google_project_default_service_accounts
+
+Allows management of a Google Cloud Platform project default service accounts.
+
+When certain Services API are enabled, Google Cloud Platform automatically creates service accounts to help to get started, but
+this is not reocmended for production environment.
+See the [Organization documentation](https://cloud.google.com/resource-manager/docs/quickstarts) for more details.
+
+## Example Usage
+
+```hcl
+resource "google_project_default_service_accounts" "my_project" {
+  project = "my-project-id"
+  action = "delete"
+}
+```
+
+To try to reactivate the default service account on the resource destroy
+
+```hcl
+resource "google_project_default_service_accounts" "my_project" {
+  project = "my-project-id"
+  action = "disable"
+  restore_policy = "REACTIVATE"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project` - (Required) The project ID. Changing this forces a new project to be created.
```

Comment:
Changing this deletes and recreates this resource, but it doesn't actually create a new project.